lib/content-services/src/lib/document-list/components/document-list.component.ts

```diff
@@ -513,6 +513,8 @@ export class DocumentListComponent extends DataTableSchema implements OnInit, On
     }
 
     ngOnInit() {
+        this.userPreferencesService.set('knowledgeRetrievalNodes', '');
```

Comment:
can we introduce this change in document-base-page.component.ts file in ACA instead of introducing this here? It would be better as it won't affect other places except alfresco.

Other question - should we even remove it for list only or maybe for every kind of page, see this issue: https://github.com/Alfresco/alfresco-content-app/pull/4147#issuecomment-2382307486